week-6/Homework/mandatory/3-slideshow/index.html

```diff
@@ -1,17 +1,73 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Slideshow</title>
-    <script src="slideshow.js"></script>
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!-- Write your HTML in here -->
-  </body>
-</html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Slideshow</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="container">
+    <div class="slideshow">
+      <img src="images/myColection.jpg" alt="myColection" />
+      <div class="text">My car collection</div>
+    </div>
+    <div class="slideshow">
+      <img src="images/Dacia 1300.jpg" alt="Dacia 1300" />
+      <div class="text">'70 Dacia 1300 </div>
+    </div>
```

Comment:
I like this car the best, but I think it's because it has more details as it's the biggest model car in the slideshow :)